lib/Dialect/OM/Evaluator/Evaluator.cpp

```diff
@@ -115,6 +116,37 @@ circt::om::Evaluator::instantiate(
   return success(std::shared_ptr<evaluator::ObjectValue>(object));
 }
 
+bool hasOMTarget(StringRef path) {
+  return (path.find("OMDontTouchedReferenceTarget") == 0 ||
```

Comment:
path.startwith would be nicer. Either way did we consider adding target kind to a path attribute as a field?